loader/src/cmd/cmdplugin.rb.in

```diff
@@ -0,0 +1,177 @@
+#!/usr/bin/ruby
+
+# Copyright (C) 2020 Open Source Robotics Foundation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# We use 'dl' for Ruby <= 1.9.x and 'fiddle' for Ruby >= 2.0.x
+if RUBY_VERSION.split('.')[0] < '2'
+  require 'dl'
+  require 'dl/import'
+  include DL
+else
+  require 'fiddle'
+  require 'fiddle/import'
+  include Fiddle
+end
+
+require 'optparse'
+
+# Constants.
+LIBRARY_NAME = '@library_location@'
+LIBRARY_VERSION = '@PROJECT_VERSION_FULL@'
+COMMON_OPTIONS =
+               "  -h [ --help ]              Print this help message.\n"\
+               "                                                    \n"        +
+               "  --force-version <VERSION>  Use a specific library version.\n"\
+               "                                                    \n"        +
+               '  --versions                 Show the available versions.'
+COMMANDS = {  'plugin' =>
+                    "Print information about plugins.\n\n"                    +
+                    "  ign plugin [options]\n\n"                              +
+                    "Options:\n\n"                                            +
+                    "  -i [ --info ]              Get info about a plugin.\n" +
+                    "                             Requires the -p option.\n"  +
+                    "\n"                                                      +
+                    "  -p [ --plugin ] arg        Name of a plugin.\n"        +
```

Comment:
I think this accepts a path to a plugin, right? "Name" could be misleading.

Reply:
good call, rephrased in c716c4b08f6bd7dfc25e0a7c6b970a0383d6190f